node/src/service/mod.rs

```diff
@@ -332,6 +337,13 @@ pub async fn start_parachain_node(
         })
     };
 
+    if parachain_config.role.is_authority() {
+        let bootstrap_config = bootstrap_config.ok_or("Could not find bootstrap config")?;
+        task_manager
+            .spawn_handle()
+            .spawn("p2p", None, run_bootstrap_node(bootstrap_config));
+    }
```

Comment:
```suggestion
    if let Ok(cfg) = bootstrap_config { 
        task_manager
            .spawn_handle()
            .spawn("p2p", None, run_bootstrap_node(bootstrap_config));
    }
```

We should be able assume that if bootstrap config is set, then the node is authority and is supposed to be the authority.